generators/form/templates/IntroductionPage.jsx.ejs

```diff
@@ -20,7 +20,7 @@ class IntroductionPage extends React.Component {
           prefillEnabled={this.props.route.formConfig.prefillEnabled}
           messages={this.props.route.formConfig.savedFormMessages}
           pageList={this.props.route.pageList}
-          startText="Start the Application"
+          startText="Start the Application">
```

Comment:
Ah, these were my bad. Somehow missed the closing bracket after removing the unnecessary props. 😓